indico/modules/events/registration/forms.py

```diff
@@ -423,3 +425,41 @@ class RejectRegistrantsForm(IndicoForm):
 
     def is_submitted(self):
         return super().is_submitted() and 'submitted' in request.form
+
+
+class RegistrationTagForm(IndicoForm):
+    """Form to create a new registration tag."""
+
+    title = StringField(_('Title'), [DataRequired()])
+    color = SelectField(_('Color'), [DataRequired()], choices=list(zip(get_sui_colors(), get_sui_colors())))
+
+    def __init__(self, *args, **kwargs):
+        self.event = kwargs.pop('event')
+        self.tag = kwargs.pop('tag', None)
+        super().__init__(*args, **kwargs)
+
+    def validate_title(self, field):
+        if not self.tag:
```

Comment:
That's not correct :/ You now don't check for collisions at all when editing... Check the logic in `EventLabelForm` for an example that handles both adding and editing.

Reply:
yup brainfart.. somehow thought I didn't have to check for collisions when editing.. :sweat_smile: